### DISCUSSION TITLE
I wanna join team sir

### DISCUSSION THREAD

**Discussion by SKetU-l** [ROM-builders/temporary#14265] [Q&A]
Hey, I wanna join the team

Github: https://github.com/SKetU-l
Telegram: https://t.me/Amngus69

Successful build:
https://cirrus-ci.com/build/5804904009695232
https://cirrus-ci.com/build/6415636791296000

Sources:X00TD
Device tree- https://github.com/SKetU-l/device_asus

Common tree- https://github.com/SKetU-l/android_device_asus_sdm660-common

Vendor tree- https://github.com/SKetU-l/vendor_asus

Kernel tree- https://github.com/SKetU-l/android_kernel_asus_sdm660  some times I am using other trees here

Cherry-pick
My bad sorry sir you can check spark branch of Device tree

Sorry sir pls reply

**Comment by SKetU-l** [comment]
https://github.com/Apon77 sir check wen?

**Reply by Apon77** [reply]
please check #learn from telegram group

**Reply by SKetU-l** [reply]
> please check #learn from telegram group

Sorry my bad I removed that cherry-picked repository you can check my device tree branch spark
